RetailCoder.VBE/UI/Command/NavigateCommand.cs

```diff
@@ -17,7 +17,7 @@ public class NavigateCommand : CommandBase, INavigateCommand
         protected override void OnExecute(object parameter)
         {
             var param = parameter as NavigateCodeEventArgs;
-            if (param == null || param.QualifiedName.Component == null)
+            if (param?.QualifiedName.Component == null)
```

Comment:
IIUC this is not quite equivalent, because if param is `null` then `param?.QualifiedName` is null, which can cause a NullReferenceException now. I think I'd keep it the old way here